docs/3.0.0-beta.x/plugins/documentation.md

```diff
@@ -171,6 +176,72 @@ To access your documentation on a custom path, you will have to update the `path
 }
 ```
 
+### Indicate which plugins documentation to generate
+
+To generate a plugin documentation, you will need to indicate the list of all the plugins for which to you wish to generate documentation. In order to do that you need to update the `pluginsForWhichToGenerateDoc` key. Leaving this key with an empty array `[]` means that not any plugin documentation will be generated. If you wish to generate documentation for all plugins, you just have to remove the key from the settings.json file.
```

Comment:
not sure but `for which to you wish` shouldn't be `for which you wish` ?

Reply:
good catch will correct it